specification/personal-demographics.yaml

```diff
@@ -299,12 +302,12 @@ paths:
       - $ref: '#/components/parameters/RequestID'
       - $ref: '#/components/parameters/CorrelationID'
       - $ref: '#/components/parameters/NHSD-End-User-Organisation-ODS'
+externalDocs:
+        description: ODS codes and documentation.
+        url: https://fhir.nhs.uk/Id/ods-organization-code    
```

Comment:
Indentation doesn't look quite right.

Reply:
Should the description and url lines be closer to the left? First time I've experienced indentation having an impact on how the code is run!

Reply:
Indentations fixed throughout and I've brought the external docs information into the body of [the main page](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) using HTML. Let me know your thoughts or if we're okay to move forward